addons/mail/tools/discuss.py

```diff
@@ -198,6 +212,10 @@ def get_result(self):
                 res[model_name] = [dict(sorted(record.items())) for record in records.values()]
         return res
 
+    def bus_send(self, notification_type="mail.record/insert", /):
+        if res := self.get_result():
+            self.target.target._bus_send(notification_type, res, subchannel=self.target.subchannel)
```

Comment:
Can't `target.target` be  `None`? I would rather an explicit error instead of letting it crash :) Maybe

```py
    assert self.target.target is not None, (
        "Missing `bus_target`. Pass it to the `Store` constructor to use `bus_send`."
    )
```